websites/A/AutoHotkey Documentation/presence.ts

```diff
@@ -0,0 +1,71 @@
+const presence = new Presence({
+  clientId: "879675566385659945"
+});
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "biglogo",
+    smallImageText: "AHK v2 Documentation",
+    startTimestamp: Math.floor(Date.now() / 1000)
+  };
+
+  switch (document.location.pathname) {
+    case "/":
+      presenceData.details = "Selecting AHK / docs version...";
+      return presence.setActivity(presenceData);
+    case "/v2/docs/AutoHotkey.htm":
+      return presence.setActivity(presenceData);
+    default:
+      presenceData.details = "Viewing a Doc page";
+      break;
+  }
+
+  /* Define the "unwanted character" regex here, cause we need that more than once XD */
+  const reg = '([^,(\\n\'"]*)',
+
+  /* Get the iframe the actual content is in.
+   This is such basic text retrieval that using the iframe data retrieval method from the documentation would be overkill... */
+   iframeObj = document.querySelector('iframe').contentDocument;
```

Comment:
Sorry, forgot this too, you should use this: https://docs.premid.app/en/dev/presence/iframe